### DISCUSSION TITLE
Solidity 101: Remix FundMe

### DISCUSSION THREAD

**Discussion by Pragadeesh-19** [Cyfrin/foundry-full-course-cu#2420] [Q&A] [answered]
I could not find the PriceConsumerV3 data feed contract for ETH/USD (Sepolia) in chainlink docs.

**Comment by EngrPips** [comment]
Hello @Pragadeesh-19, Why not check the official chainlink repo instead?

**Comment by igorlnunes** [comment, marked as answer]
@Pragadeesh-19, 

There have been updates to the Chainlink documentation, including changes to the Interface and DataFeed code. Issue #2303  was referenced to obtain the ETH/USD price.

-> Current code: https://docs.chain.link/data-feeds/using-data-feeds#solidity

1) Change the BTC/ETH address to the USD/ETH Sepolia Testnet address found at this link: https://docs.chain.link/data-feeds/price-feeds/addresses?network=ethereum&page=1#sepolia-testnet;
2) In the Remix IDE, compile the code with the new address;
3) Deploy the contract;
4) The getChainlinkDataFeedLatestAnswer call button will return the value for the pair.

Let me know if you'd like any adjustments! I hope I was able to help.

**Reply by Pragadeesh-19** [reply]
Thank you, Now its working!!

**Reply by xi2And33** [reply]
so the main point the lesson want to tell us is we can use different address to get the pair of the different price?

**Reply by EngrPips** [reply]
You can draw that conclusion.

**Comment by EngrPips** [comment]
Interesting
